src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int randomNumber = new Random().nextInt(100);
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Lottery().randomNumber);
```

Comment:
you don't need to create a Lottery object, you can use the field `randomNumber` directly